### ISSUE TITLE
Database contains creation metadata

### ISSUE THREAD

**Issue by Stoom** [Stoom/KeePass-TDD#6]
```gherkin
Scenario: A new database is created and includes metadata about its creation.

Given a database is created
Then the name changed time is set to creation time.
And the description changed time is set to creation time.
And the master key changed time is set to creation time.
And the master key expiration warning is infinite.
And the master key expiration is infinite.
```

**Comment by Stoom** [comment]
Fixed in commit 86fe95e

**Comment by Stoom** [comment]
Fixed in commit 86fe95e
